src/test/java/net/testudobank/tests/MvcControllerIntegTest.java

```diff
@@ -189,6 +189,54 @@ public void testSimpleWithdraw() throws SQLException, ScriptException {
     MvcControllerIntegTestHelpers.checkTransactionLog(customer1TransactionLog, timeWhenWithdrawRequestSent, CUSTOMER1_ID, MvcController.TRANSACTION_HISTORY_WITHDRAW_ACTION, CUSTOMER1_AMOUNT_TO_WITHDRAW_IN_PENNIES);
   }
 
+  @Test
```

Comment:
Write a comment before the integration test explaining what you'll be testing for.